### ISSUE TITLE
Deployment package doesn't include native dependencies from NuGet packages, even though they show in Application Files.

### ISSUE THREAD

**Issue by lucent-sea** [dotnet/wpf#3928]
* .NET Core Version: (e.g. 3.0 Preview1, or daily build number, use `dotnet --info`)
```
.NET SDK (reflecting any global.json):
 Version:   5.0.101
 Commit:    d05174dc5a

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19042
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\5.0.101\

Host (useful for support):
  Version: 5.0.1
  Commit:  b02e13abab

.NET SDKs installed:
  2.2.207 [C:\Program Files\dotnet\sdk]
  3.0.100 [C:\Program Files\dotnet\sdk]
  5.0.101 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.All 2.1.23 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.2.8 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.23 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.2.8 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.0.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.10 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.1 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.23 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.8 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.2 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.10 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.1 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.1.2 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.1.10 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.1 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```
* Windows version: (`winver`)
`Version 20H2 (OS Build 19042.685)
* Does the bug reproduce also in WPF for .NET Framework 4.8?: Don't know.
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? If yes, please file the issue via the instructions [here](https://docs.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio?view=vs-2019).
* Security issues and bugs should be reported privately, learn more via our [responsible disclosure guidelines](https://github.com/dotnet/wpf/blob/master/README.md#reporting-security-issues-and-security-bugs).

 
 
 **Problem description:**
 I'm using the [MixedReality WebRTC library.](https://github.com/microsoft/MixedReality-WebRTC).  When I build a ClickOnce profile, the native library `mrwebrtc.dll` shows in the list of Application Files to include.  But when I publish the application and install it, it's missing.  And of course, the application doesn't run because it can't find it.

![image](https://user-images.githubusercontent.com/20995508/103137120-098ba300-467b-11eb-9857-081bc5df056e.png)

 **Actual behavior:** 
 The installation folder is missing the native dependency `mrwebrtc.dll`.
 **Expected behavior:**
The native dependency should be included in the deployment and copied to the installation folder.
 
 **Minimal repro:**
* Create WPF project.
* Install MixedReality WebRTC package: `Install-Package Microsoft.MixedReality.WebRTC -Version 2.0.2`
* Create a ClickOnce publish profile and note that `mrwebrtc.dll` is included in Application Files.
* Publish to a folder and install.
* Notice that the library is not included in the installation directory.

**Comment by ryalanms** [comment]
@dsplaisted: Who owns ClickOnce publishing?  Thanks.

**Comment by dsplaisted** [comment]
@MSLukeWest @John-Hart Can you route this to the right folks to look at a ClickOnce publishing issue?

**Comment by sujitnayak** [comment]
A fix for this will be included in the upcoming service release for 16.8.

**Comment by lucent-sea** [comment]
It looks like this is fixed now.  Thank you!  :)
